sky/cli.py

```diff
@@ -3476,6 +3485,15 @@ def storage_delete(names: List[str], all: bool):  # pylint: disable=redefined-bu
         names = [s['name'] for s in storages]
     else:
         names = _get_glob_storages(names)
+    if not yes:
+        storage_names = ', '.join(names)
+        storage_str = 'storages' if len(names) > 1 else 'storage'
+        click.confirm(
+            f'Deleting {len(names)} {storage_str}: '
+            f'{storage_names}. Proceed?',
+            default=True,
+            abort=True,
+            show_default=True)
```

Comment:
We should exit if no storages are found, but a prompt is shown:
```
(base) ➜  ~ sky storage delete abc
Storage abc not found.
Deleting 0 storage: . Proceed? [Y/n]: 
```

Reply:
I've changed the `if not yes` if statement to check for `names` if it has length zero. the subprocess util should take care of the rest if nothing is found.